r/flightsql/gizmosql/README.md

```diff
@@ -0,0 +1,87 @@
+<!--
+Copyright 2025 Columnar Technologies Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Connecting Go and GizmoSQL (an Arrow Flight SQL server - powered by DuckDB or SQLite) with ADBC
+
+## Instructions
+
+This example uses [GizmoSQL](https://gizmodata.com/gizmosql), but other open source tools and vendor products that support Arrow Flight SQL will also work with this driver.
+
+> [!TIP]
+> If you already have a GizmoSQL instance running, skip the steps to set up GizmoSQL.
+
+### Prerequisites
+
+1. [Install dbc](https://docs.columnar.tech/dbc/getting_started/installation/)
+
+1. Install R packages `adbcdrivermanager`, `arrow`, and `tibble`:
+
+   ```r
+   install.packages(c("adbcdrivermanager", "arrow"))
+   install.packages("tibble")
+   ```
```

Comment:
@ianmcook I was mistaken, but tibble is not a required dependency of arrow, so it seems that an error occurs if we do not explicitly install it.
I will update the documentation later.